internal/files/controllers/siteconfig-template-cm.yaml

```diff
@@ -0,0 +1,164 @@
+kind: ConfigMap
+metadata:
+  name: default-siteconfig-template
+  namespace: default
```

Comment:
Should this be in the oran-o2ims namespace instead?

Reply:
Yes, this will be changed in the next PR. Just put it in the default temporarily since it was unclear if it will be shipped with IMS operator.